compiler/resolution/functionResolution.cpp

```diff
@@ -11746,14 +11747,54 @@ static void postResolveLiftStaticVars() {
       // Move the definition point of the wrapper into the module scope.
       // But first, keep a handle on the block to be able to move it later.
       auto wrapperBlock = toBlockStmt(wrapperSym->defPoint->parentExpr);
-      INT_ASSERT(wrapperBlock);
-      call->getModule()->block->insertAtHead(wrapperSym->defPoint->remove());
+      targetMod->block->insertAtHead(wrapperSym->defPoint->remove());
+
+      // If we created some cleanup code to run in the per-locale case,
+      // retrieve it now.
+      BlockStmt* cleanupBlock = nullptr;
+      if (wrapperSym->hasFlag(FLAG_LOCALE_PRIVATE)) {
+        cleanupBlock = toBlockStmt(wrapperBlock->body.tail->remove());
+        INT_ASSERT(wrapperBlock);
+      }
 
       // Now move the initialization code into the module init function.
       // The last statement is the 'return void', which we keep.
-      call->getModule()->initFn->body->insertAtHead(wrapperBlock->remove());
+      targetMod->initFn->body->insertAtHead(wrapperBlock->remove());
       wrapperBlock->flattenAndRemove();
       call->remove();
+
+      // In some cases, we need to do cleanup. See the comment
+      // in normalize.cpp's preNormalizeHandleStaticVars for the
+      // structure of the cleanup block.
+      if (!cleanupBlock) continue;
+      SET_LINENO(wrapperSym);
+
+      auto cleanupFnDefBlock = toBlockStmt(cleanupBlock->body.head->remove());
+      INT_ASSERT1(cleanupFnDefBlock);
```

Comment:
What is `INT_ASSERT1`? Just a typo? TIL, why use over `INT_ASSERT`?

Reply:
It is a typo... but... it compiled? Huh.